src/mrack/providers/openstack.py

```diff
@@ -709,7 +714,7 @@ def prov_result_to_host_data(self, prov_result, req):
         result = {}
 
         result["id"] = prov_result.get("id")
-        result["name"] = prov_result.get("name")
```

Comment:
Just a note, that this hides the actual name got/requested from OpenStack.

Reply:
Correct, but that's the goal, isn't it? It is more important to have in mrack db the actual name we use as a reference in all the parts of mrack. The name in the provider is not important - used for anything as we use provider ID for operations with the host in the provider.